### PR TITLE
Added Server->broadcastTip() and Server->broadcastPopup()

### PR DESCRIPTION
**Pull request by Striker209**

### CODE REVIEW

File: src/pocketmine/Server.php [hunk 1]
```diff
@@ -1766,6 +1766,40 @@ public function broadcastMessage($message, $recipients = null){
 	}
 
 	/**
+	 * @param               $message
```
Comment (PEMapModder) [inline src/pocketmine/Server.php]: Why no type?

**Comment by TruDan** [comment]
:+1:

File: src/pocketmine/Server.php (hunk 1 above)
Comment (Striker209MPE) [inline src/pocketmine/Server.php]: I did the same as the Server->broadcastMessage() method but yes it should :D

**Comment by sekjun9878** [comment]
Merged with adc5aa11ac26ae0668368f05dc01e19aae605b33. Thank you for the PR!

**Pull request closed without merging**